docs/USERGUIDE.md

```diff
@@ -0,0 +1,74 @@
+# User Guide
+
+## Design
+
+Hypersds-operator manages a custom resource (CR), `CephCluster` to manage an external ceph cluster which is located outside of Kubernetes cluster. Hypersds-operator watches `CephCluster` CR to create hypersds-provisioner which installs and updates the external ceph cluster.
+
+![hypersds-operator project overall diagram](assets/overall.png)
+
+## Workflow
```

Comment:
The workflow part need to be updated.
It also need role and rolebinding for provisioner pod. The image has been updated.